### PR TITLE
:sparkles: Add details Lesson

### PR DESCRIPTION
**Pull request by miicolas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "framer-motion": "^11.2.10",
     "fs": "^0.0.1-security",
     "gray-matter": "^4.0.3",
+    "gsap": "^3.12.5",
```
Comment (miicolas) [inline package.json]: delete gsap

File: src/app/content/lessons/decouvrir-git.mdx [hunk 2]
```diff
@@ -2,10 +2,16 @@
 title : Découvrir Git
 slug : decouvrir-git
 publishedAt: "2024-03-01"
+repo: [{label: "Repo GitHub", url: "https://github.com/julien-sassenrath/git-course"}]
```
Comment (miicolas) [inline src/app/content/lessons/decouvrir-git.mdx]: update repo & link

File: src/app/content/lessons/decouvrir-git.mdx [hunk 3]
```diff
@@ -2,10 +2,16 @@
...
+link : [{label: "Vidéo", url: "https://www.youtube.com/watch?v=HVsySz-fAg8"}, {label: "Vidéo", url: "https://www.youtube.com/watch?v=HVsySz-fAg8"}]
 
 ---
 
-
+```js {2}showLineNumbers
```
Comment (miicolas) [inline src/app/content/lessons/decouvrir-git.mdx]: delete exemple

File: src/app/globals.css [hunk 4]
```diff
@@ -178,3 +178,22 @@ body[data-scroll-locked]{
...
+  background-color: rgba(30, 41, 59, .7) !important;
+  padding: 0.2rem 0.5rem !important;
+  color: green !important;
+  margin: -0.2rem -0.5rem !important;
+  position: absolute;
+  width: 100%;
+  transition: all 0.3s ease-in-out;
+}
+.highlighted-code-line:hover {
+
```
Comment (miicolas) [inline src/app/globals.css]: optimisation code

File: src/components/sheet/sheetLessons.tsx [hunk 5]
```diff
@@ -116,11 +116,11 @@ export default async function SheetLessons({ userId }: userIdProps) {
                         (lesson: LessonWithFormattedSortNumber) => (
                           <li
                             key={lesson.id}
-                            className="group flex items-center gap-2 hover:bg-torea-50/10 transition-all duration-300 px-2 py-1 rounded-md"
+                            className="group flex items-cneter gap-2 hover:bg-torea-50/10 transition-all duration-300 px-2 py-1 rounded-md"
```
Comment (miicolas) [inline src/components/sheet/sheetLessons.tsx]: fix typing error

File: src/lib/lesson.ts [hunk 6]
```diff
@@ -31,6 +37,7 @@ export async function markdownToHTML(markdown: string) {
       keepBackground: false,
     })
     .use(rehypeStringify)
+
```
Comment (miicolas) [inline src/lib/lesson.ts]: optimisation

File: src/components/sheet/sheetLessons.tsx (hunk 5 above)
Comment (Drixares) [reply src/components/sheet/sheetLessons.tsx]: items-cneter ?

**Review by Drixares — COMMENTED** [review]
Carré

**Review by miicolas — COMMENTED** [review]
Fix error done

**Review by miicolas — COMMENTED** [review]
Details Lesson Done

**Pull request merged**